### DISCUSSION TITLE
11월 1주차

### DISCUSSION THREAD

**Discussion by healtheloper** [Co-Studo/Co-Studo-front#73] [Weekly meeting]
### 컴포넌트 인터페이스 결정 방식
- [issue 에 같이 등록](https://github.com/Co-Studo/Co-Studo-front/issues/67)
- Dony 궁금한점: issue에 등록한 것과 다르게 변경되면 issue도 닫기 전에 업데이트 해줘야하는가?

### Styled Components vs Emotion
- Styled Components의 css prop 방식은 as 사용이 번거롭다

### PR, dicussion 확인
- PR, discussion이 올라오면 바로 확인
- PR 닫는 기준 설정

**Comment by jindonyy** [comment]
[Notion 회의록](https://costudo.notion.site/11-1-1300f05488d440babeb81b0f9bb5de78)
